src/core/service.c

```diff
@@ -2466,7 +2468,7 @@ static void service_enter_condition(Service *s) {
                 r = service_spawn(s,
                                   s->control_command,
                                   s->timeout_start_usec,
-                                  EXEC_APPLY_SANDBOXING|EXEC_APPLY_CHROOT|EXEC_IS_CONTROL|EXEC_APPLY_TTY_STDIN,
+                                  EXEC_IS_CONTROL|EXEC_APPLY_TTY_STDIN,
```

Comment:
Here, EXEC_WRITE_CREDENTIALS is missing.

Reply:
We don't want `ExecCondition=` to access credentials IIRC. Generally it should be pretty irrelevant from other service processes.

Reply:
Then, I may misunderstand the flags in `service_run_next_control()`.